frontend/src/lib/utils/scoreUtils.ts

```diff
@@ -8,7 +8,7 @@ export const calculateAverageScore = (scoreSum: number, totalSessions: number):
     return '-';
 
   const averageScorePercentage = Math.round(scoreSum / totalSessions);
-  const averageScore = ((averageScorePercentage / 100) * 5).toFixed(1);
+  const averageScore = ((averageScorePercentage / 100) * 20).toFixed(1);
```

Comment:
We can remove this calculation entirely. The average score is just scoreSum/totalSessions. If we have 2 sessions, one with a score of 10 and another with a score of 20, we woudl do 30/2=15 to get the average score. No percentage calculation necessary.